src/utils/token.ts

```diff
@@ -18,16 +21,10 @@ export function fetchTokenSymbol(
   const symbolResult = contract.try_symbol()
   if (symbolResult.reverted) {
     const symbolResultBytes = contractSymbolBytes.try_symbol()
-    if (!symbolResultBytes.reverted) {
+    if (!symbolResultBytes.reverted && !isNullEthValue(symbolResultBytes.value.toHexString())) {
       // for broken pairs that have no symbol function exposed
       if (!isNullEthValue(symbolResultBytes.value.toHexString())) {
         symbolValue = symbolResultBytes.value.toString()
-      } else {
-        // try with the static definition
-        const staticTokenDefinition = getStaticDefinition(tokenAddress, staticTokenDefinitions)
```

Comment:
we moved these blocks to the top of the function so the priority is 
1. static token definitions 
2. contract call
3. contract bytes call